### PR TITLE
[Outlook][Add img] Added img to internet headers article

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9c5600a](https://github.com/OfficeDev/office-js-docs-pr/commits/9c5600aa4809b501f5ed5526daaaf229aa5ee7ba)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/outlook-internet-headers.png](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-imgintheaders/docs/images/outlook-internet-headers.png) | :white_check_mark:Succeeded | |
[docs/outlook/internet-headers.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-imgintheaders/docs/outlook/internet-headers.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004101834318737-1738%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=0Y8n%2FeXFHDKjsfnGuOiOWjgF4IKSyJNW7EiKxcdzmkA%3D&st=2020-04-10T18%3A30%3A40Z&se=2020-05-11T18%3A35%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/outlook/internet-headers.md [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 ---
 
 # Get and set internet headers on a message in an Outlook add-in
 
 ## Background
 
-A common requirement in Outlook add-ins development is to store custom properties associated with an add-in at different levels. At present, custom properties are stored at the item or mailbox level.
+A common requirement in Outlook add-ins development is to store custom properties associated with an add-in at different levels. 
+
+![Diagram of internet headers. Text: User 1 sends email. Add-in manages custom internet headers while user is composing email. User 2 receives the email. Add-in gets internet headers from received email then parses and uses custom headers. ](../images/outlook-internet-headers.png)
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/internet-headers.md:14]: The original location in the first commit is correct.

File: docs/outlook/internet-headers.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
 ---
 title: Get and set internet headers
 description: How to get and set internet headers on a message in an Outlook add-in.
-ms.date: 11/04/2019
+ms.date: 04/10/2019
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/internet-headers.md:4]:
```suggestion
ms.date: 04/10/2020
```

File: docs/outlook/internet-headers.md [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
 localization_priority: Normal
 ---
 
 # Get and set internet headers on a message in an Outlook add-in
```
Comment (ElizabethSamuel-MSFT) [inline docs/outlook/internet-headers.md:9]: Alternatively, you can add an introductory sentence then insert the image after that.
```suggestion
The ability to set custom headers on an email then read them from the recipient's side enables metadata to be shared between users.

![Diagram of internet headers. Text: User 1 sends email. Add-in manages custom internet headers while user is composing email. User 2 receives the email. Add-in gets internet headers from received email then parses and uses custom headers. ](../images/outlook-internet-headers.png)

```

File: docs/outlook/internet-headers.md (hunk 1 above)
Comment (mscharlock) [reply docs/outlook/internet-headers.md:14]: Yeah I debated about where to put it, I think I will move it back to where I had it the first time.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[233670a](https://github.com/OfficeDev/office-js-docs-pr/commits/233670a53aabbc9427ca82f43bcd191b82eab824)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/outlook-internet-headers.png](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-imgintheaders/docs/images/outlook-internet-headers.png) | :white_check_mark:Succeeded | |
[docs/outlook/internet-headers.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-imgintheaders/docs/outlook/internet-headers.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004102232017561-1738%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=coJ0vZmnQ8Vcj5MyqalVk4pf3vDTMEXooPQvt55uwOY%3D&st=2020-04-10T22%3A28%3A18Z&se=2020-05-11T22%3A33%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fdf764d](https://github.com/OfficeDev/office-js-docs-pr/commits/fdf764ddad481dc717459dc563ceec67eedcfa7c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/outlook-internet-headers.png](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-imgintheaders/docs/images/outlook-internet-headers.png) | :white_check_mark:Succeeded | |
[docs/outlook/internet-headers.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-imgintheaders/docs/outlook/internet-headers.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004102256511795-1738%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=DSLNgH8fDCP8NWBqhPV7jDQJCKLb9DYlJnjMBTJAtQs%3D&st=2020-04-10T22%3A53%3A02Z&se=2020-05-11T22%3A58%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**